kolibri/plugins/learn/assets/src/views/classes/Hint.vue

```diff
@@ -1,59 +0,0 @@
-<template>
```

Comment:
I wonder if we could just leave this as is, and instead remove the hint button inside the Perseus Renderer, and then just hook things up via props and events via the `LessonMasteryBar`?

That would leave us open for reusing this in say, the QTI renderer if needed.

Reply:
I'd rather not keep the cruft around in case it's maybe useful. The component isn't usable as-is and is only used in `LessonMasteryBar` so I don't see the need for a reusable component - particularly if the QTI Renderer ends up finding `LessonMasterBar` useful in its design, in which case the hint code will be reused as-is.